lambdas/verifiable-credential/types/verifiable-credentials.ts

```diff
@@ -4,17 +4,20 @@ export enum ChronoUnit {
   Hours = "hours",
   Days = "days",
 }
-
+export enum VC_CONTEXT {
+  W3_BASE_CONTEXT = "https://www.w3.org/2018/credentials/v1",
+  DI_CONTEXT = "https://vocab.london.cloudapps.digital/contexts/identity-v1.jsonld",
+}
```

Comment:
Is it worth having one high level types folder? Instead of multiple sub folders? Follow the common-lambdas pattern

Reply:
I don't know really, I suppose for consistency i'll move to the types folder. My thinking was that things the work together as a unit can have a types directory which  could be a library that is self contained and a package of it's own. For instance I originally had the `verifiable-credential/types` folder having subject.ts and the other one. The builder can be used by multiple projects. However I discovered the subject didn't really belong and seems to always be the responsibility of the actual credential issue i.e (toy, address, kbv) all handle the subject differently

Reply:
That makes sense, definitely something to think about when we do the common TS lib - do we want one types lib or have lots of function libs each with their own types folder (think I prefer that)